src/ch/blinkenlights/android/vanilla/LibraryPagerAdapter.java

```diff
@@ -471,6 +466,7 @@ public void clearLimiter(int type)
 				mPendingFileLimiter = null;
 			} else {
 				mFilesAdapter.setLimiter(null);
+				loadSortOrder(mFilesAdapter);
```

Comment:
Ehhm....
I thought the files adapter must be updated in the same way as the media adapters. But actually I dunno why.
Removed.